lib/board.rb

```diff
@@ -0,0 +1,137 @@
+require 'terminal-table'
+require 'colorize'
+
+module Scrabble
+  class Board
+
+    BOARD_SIDE_SIZE = 15
```

Comment:
I'm really happy that it was an obvious choice to make this a constant, instead of hardcoding a size